gemrb/core/GameScript/GSUtils.cpp

```diff
@@ -522,6 +522,11 @@ void DisplayStringCore(Scriptable* const Sender, int Strref, int flags)
 		unsigned int len = 0;
 		core->GetAudioDrv()->Play(Sound, channel, 0, 0, speech, &len);
 		ieDword counter = ( AI_UPDATE_TIME * len ) / 1000;
+
+		if (len > 0) {
+			reinterpret_cast<Actor*>(Sender)->SetAnimatedTalking(len);
```

Comment:
There's no guarantee that the Scriptable is of Actor type, so please add a check first: Sender->Type == ST_ACTOR

Reply:
Oh yeah, I was fooled by that line above which is not formally a barrier, I guess: https://github.com/gemrb/gemrb/blob/23ae06a166caa3e5723fc82fc119955b31260f8b/gemrb/core/GameScript/GSUtils.cpp#L461

Reply:
Fixed that.